### PR TITLE
Parallelized Elasticsearch request for serving summary response (fixes #467)

### LINKED ISSUE #467: Parallelize ES requests for `/repository/summary`
Following up on #460 , we should parallelize the three ES requests that we do for /repository/summary

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/AZUL-311)
┆Project Name: azul
┆Issue Number: AZUL-311

### PR DESCRIPTION
**Pull request by achave11-ucsc**
Threaded Elasticsearch requests for serving `repository/summary` faster.

### CODE REVIEW

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
One nit, see below. 

No periods at the end of the commit title. Second commit title is too long. Abbreviate ES. s/parallel/parallelize/ like you did in the PR title.

File: lambdas/service/app.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 app = Chalice(app_name=config.service_name, configure_logs=False)
 app.debug = True  # FIXME: this should be configurable via environment variable (https://github.com/DataBiosphere/azul/issues/419)
-app.log.setLevel(logging.DEBUG)  # FIXME: please use module logger instead (https://github.com/DataBiosphere/azul/issues/419)
+app.log.setLevel(
```
[the remark this answers is not in the document]
Comment (achave11) [reply lambdas/service/app.py]: Automated wrapping by PyCharm.

**Review by hannes-ucsc — APPROVED** [review]

**Comment by achave11** [comment]
<3

**Comment by hannes-ucsc** [comment]
Before I merge this I would like to see this in action. For tomorrow, could you have an instance ready (either deployed remotely or `make local`) for me to look at?

**Comment by achave11** [comment]
> Before I merge this I would like to see this in action. For tomorrow, could you have an instance ready (either deployed remotely or `make local`) for me to look at?

Will do!

**Pull request merged**